pkg/descheduler/strategy_migration.go

```diff
@@ -151,4 +152,67 @@ var pluginsMap = map[string]func(ctx context.Context, nodes []*v1.Node, params *
 			klog.V(1).ErrorS(err, "plugin finished with error", "pluginName", removeduplicates.PluginName)
 		}
 	},
+	"HighNodeUtilization": func(ctx context.Context, nodes []*v1.Node, params *api.StrategyParameters, handle *handleImpl) {
+		var (
+			thresholds       = make(componentconfig.ResourceThresholds)
+			targetThresholds = make(componentconfig.ResourceThresholds)
+		)
+		for key, value := range params.NodeResourceUtilizationThresholds.Thresholds {
+			thresholds[key] = componentconfig.Percentage(value)
+		}
+		args := &componentconfig.HighNodeUtilizationArgs{
+			ThresholdPriority:          *params.ThresholdPriority,
+			ThresholdPriorityClassName: params.ThresholdPriorityClassName,
+			Thresholds:                 thresholds,
```

Comment:
Passing `params.NodeResourceUtilizationThresholds.Thresholds` directly instead of `thresholds` did not work?